src/api/user-api.js

```diff
@@ -0,0 +1,19 @@
+import config from '~/config';
+import $ from 'jquery';
+
+export function login(username, password) {
+    const payload = {
+        email: username,
+        password,
+    };
+    return $.post(`${config.backend}/authentication/login`, payload);
+}
+
+export function checkAuth() {
+    const token = localStorage.getItem('auth_token');
```

Comment:
Note to self: verify if `localStorage` is the same thing as cookies and if it is a secure place to store this.